recipes/moff/meta.yaml

```diff
@@ -1,12 +1,14 @@
+{% set version = "v1.2" %}
```

Comment:
I think it is better praxis to call it 1.2 and add "v" in front of it in the url.

Reply:
Changed!